llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -752,6 +752,60 @@ static bool removeSYCLKernelsConstRefArray(Module &M) {
   return true;
 }
 
+SmallVector<module_split::ModuleDesc, 2>
+handleESIMD(module_split::ModuleDesc &&MDesc, bool &Modified,
+            bool &SplitOccurred) {
+  // Do SYCL/ESIMD splitting. It happens always, as ESIMD and SYCL must
```

Comment:
Reword comment: "Do splitting based on ESIMD markings. It happens always, as ESIMD and non-ESIMD code must undergo different set of LLVM IR passes.
I think it will be better to call the two parts as ESIMD and non-ESIMD instead of ESIMD and SYCL.

This suggestion is based on my understanding. Pls feel free to let me know if I am wrong here.

Thanks